internal/broadcaster/utxo_creator.go

```diff
@@ -182,7 +199,6 @@ func (b *UTXOCreator) splitOutputs(requestedOutputs int, requestedSatoshisPerOut
 			return nil, err
 		}
 
-		// only split if splitting increases nr of outputs
```

Comment:
We shouldn't remove such comments as they can facilitate the understanding of the code

Reply:
I thought i pushed the change, somehow it went missing pushed again